playground/css/vite.config.js

```diff
@@ -1,5 +1,10 @@
 const path = require('node:path')
 
+// trigger scss bug: https://github.com/sass/dart-sass/issues/710
+// make sure Vite handles safely
+globalThis.window = {}
+globalThis.location = new URL('http://localhost/')
```

Comment:
FYI when `navigator` is shimmed (`globalThis.navigator = {}`), `sass` crashes. (https://github.com/vitejs/vite/issues/5815#issuecomment-1113863890)
This is not directly related to this PR, but just in case you hit this in future.